pennylane/transforms/specs.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Code for resource estimation"""
+
+
+def specs(qnode, max_expansion=None):
+    """Resource information about a quantum circuit.
+
+    This transform performs a tape construction without execution in order to access ``QNode.specs``.
```

Comment:
```suggestion
    This transform converts a QNode into a callable that provides resource information
    about the circuit.
```
Maybe? I didn't find the above sentence too clear.